src/components/Cluster/Cluster.scss

```diff
@@ -20,11 +20,22 @@ limitations under the License.
   display: flex;
   width: 100%;
   align-items: center;
-  padding: 1em 2em;
-  .clusterIcon {
-    margin-right: 10px;
+  .fa {
+    color: $kubeapps-text !important;
   }
-  .clusterField {
-    font-size: 1.4em !important;
+  .clusterUrl {
+    text-align: left;
+    label {
+      margin: 0;
+    }
+    input {
+      font-size: 1.3em !important;
```

Comment:
Have you tried the `type-*` modifiers?

Reply:
Oh I missed that helper in the doc, good catch 👍